game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -937,7 +933,7 @@ private void notifyListeners() {
     private final UiContext uiContext;
     private final UnitCategory category;
     private final ScrollableTextField textField;
-    private final List<WidgetChangedListener> listeners = new CopyOnWriteArrayList<>();
+    private final List<Runnable> listeners = new ArrayList<>();
```

Comment:
I verified the `addChangeListener()` method is only called from the EDT, so a thread-safe collection is not necessary.